src/Umbraco.Web/Telemetry/ReportSiteTask.cs

```diff
@@ -0,0 +1,128 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Umbraco.Core;
+using Umbraco.Core.Configuration;
+using Umbraco.Core.IO;
+using Umbraco.Core.Logging;
+using Umbraco.Web.Scheduling;
+
+namespace Umbraco.Web.Telemetry
+{
+    public class ReportSiteTask : RecurringTaskBase
+    {
+        private IRuntimeState _runtime;
+        private IProfilingLogger _logger;
+        private static HttpClient _httpClient;
+
+        public ReportSiteTask(IBackgroundTaskRunner<RecurringTaskBase> runner, int delayBeforeWeStart, int howOftenWeRepeat, IRuntimeState runtime, IProfilingLogger logger)
+            : base(runner, delayBeforeWeStart, howOftenWeRepeat)
+        {
+            _runtime = runtime;
+            _logger = logger;
+
+            if (_httpClient == null)
```

Comment:
It will always be null in a ctor

Reply:
Yep makes sense - will just new it up & remove the if null check

It was a case of me looking at/following a similar use-case from KeepAlive background task
https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Web/Scheduling/KeepAlive.cs#L20-L29